tachyon/math/polynomials/univariate/radix2_evaluation_domain.h

```diff
@@ -214,26 +209,23 @@ class Radix2EvaluationDomain : public UnivariateEvaluationDomain<F, MaxDegree>,
     size_t duplicity_of_initials = size_t{1} << (log_n - log_d);
     evals.evaluations_.resize(n, F::Zero());
     this->SwapElements(evals, num_coeffs, log_n);
-    if (duplicity_of_initials > 1) {
+    size_t start_gap = 1;
+    // NOTE(GideokKim): |base::ParallelizeByChunkSize(container, chunk_size,
+    // callback)| is not called when |duplicity_of_initials| is 2 or 3. Since we
+    // have not benchmarked this case, it is called only when it is 4 or higher.
```

Comment:
I think simplifying the full statement to this would be better; however, I'm not sure if this comment is needed, since users can go see that `kDegreeAwareFFTThresholdFactor` is 4.
```suggestion
    // NOTE(GideokKim): |base::ParallelizeByChunkSize(container, chunk_size,
    // callback)| is only called when |duplicity_of_initials| is 4 or higher.

Reply:
I think the same thing. If we are just going to write that it's called only when `duplicity_of_initials` is 4 or higher, I don't think this comment is needed.